controllers/postController.js

```diff
@@ -0,0 +1,76 @@
+const Post = require('../models/Post');
+
+const deletePost = (req, res) => {
+  const { postId } = req.params;
+
```

Comment:
postId should just be id because we would be in the post function so it would be redundant to have Post.deletePost(postId) when it should Post.deletePost(id)